docs/how-to/code/build-docs/task.yaml

```diff
@@ -1,34 +0,0 @@
-###########################################
-# IMPORTANT
-# Comments matter!
-# The docs use the wrapping comments as 
-# markers for including said instructions 
-# as snippets in the docs.
-###########################################
-summary: test the "Build the docs" guide
```

Comment:
Why did this get deleted?

Reply:
But this test verifies that the [documentation on how to build the docs](https://canonical-rockcraft.readthedocs-hosted.com/en/latest/how-to/build-docs/) is working. It's used by people that want to contribute docs, etc, so it needs to be updated

Reply:
Well, then I also need to update the docs, because the command now is `tox -e build-docs`

Reply:
yeap

Reply:
That's the point of these tests; if you update the test, the documentation will update automatically because it consumes lines from this task.yaml.

Reply:
Updated